packages/bootstrap-4/src/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -91,8 +90,7 @@ const CheckboxesWidget = ({
                 onChange={_onChange(option)}
                 onBlur={_onBlur}
                 onFocus={_onFocus}
-                disabled={disabled || itemDisabled || false}
-                readOnly={readonly}
+                disabled={disabled || readonly || itemDisabled || false}
```

Comment:
Let's keep the same order as in core:

https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/core/src/components/widgets/CheckboxesWidget.js#L33

```suggestion
                disabled={disabled || itemDisabled || readonly}
```